Content.Server/Database/ServerDbBase.cs

```diff
@@ -901,7 +901,7 @@ public async Task AddAdminLogs(List<AdminLog> logs)
                     if (attempt >= maxRetryAttempts)
                     {
                         _opsLog.Error($"Max retry attempts reached. Failed to save {logs.Count} admin logs.");
-                        return;
+                        throw;
```

Comment:
Isn't this going to crash the dev client? Why not just log an Error for the game too, and then return